js/models/certification.ts

```diff
@@ -3,12 +3,21 @@ import { HeavyRailLine } from "../types";
 import { DateTime } from "luxon";
 import { z } from "zod";
 
+const CERT_TYPES = ["right_of_way", "rail"];
+
 export type Certification = {
   type: string;
   railLine: HeavyRailLine;
   expires: DateTime;
 };
 
+export type MissingCertification = Omit<Certification, "expires">;
```

Comment:
This type is a bit weird in that unlike Certification, it doesn't describe data that comes from the server, it's data that the frontend creates during its calculations and then sends to the backend.

Is that worth calling out in a `/** doc comment */`?